### PR TITLE
Develop 2 Master

### PR DESCRIPTION
**Pull request by matinlotfali**
This pull request attempts to merge the 'develop' into 'master' as a start of a release submission.

### CODE REVIEW

File: .github/workflows/build-yocto.yml [hunk 1]
```diff
@@ -21,16 +21,11 @@ jobs:
     - name: Build the Docker image
       run: cd Build && ./build.sh; 
     
-    - name: Run the Docker image and build output files with SDK
```
Comment (mistyron) [inline .github/workflows/build-yocto.yml:24]: Did we want to build the SDK as part of a release too though?

**Review by mistyron — COMMENTED** [review]
Please check my comment regarding the SDK

File: .github/workflows/build-yocto.yml (hunk 1 above)
Comment (matinlotfali) [reply .github/workflows/build-yocto.yml:24]: I don't think so

**Review by mistyron — APPROVED** [review]
LGTM

**Pull request merged**